apps/mull-ui/src/app/components/chat-bubble-list/chat-bubble-list.scss

```diff
@@ -0,0 +1,3 @@
+.img {
```

Comment:
💡 Remember that the styling you're defining here is global, and will affect anything that has `img` as a class. I suggest having a more specific name, something like `chat-bubble-image` to avoid clashes

Reply:
Noticed this too. 100% agree